image_width: 1280
image_height: 720
camera_name: my_cam_720p
camera_matrix:
  rows: 3
  cols: 3
  data: [903.4205, 0.0, 648.2501,
         0.0, 903.7381, 380.6823,
         0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.1006, -0.2574, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0,
         0.0, 1.0, 0.0,
         0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [903.4205, 0.0,   648.2501, 0.0,
         0.0,   903.7381, 380.6823, 0.0,
         0.0,     0.0,       1.0,  0.0]

